node/network/protocol/src/request_response/mod.rs

```diff
@@ -126,48 +126,53 @@ impl Protocol {
 	///
 	/// Returns a receiver for messages received on this protocol and the requested
 	/// `ProtocolConfig`.
-	pub fn get_config(self) -> (mpsc::Receiver<network::IncomingRequest>, RequestResponseConfig) {
-		let p_name = self.into_protocol_name();
+	pub fn get_config<Hash: AsRef<[u8]>>(
+		self,
+		genesis_hash: &Hash,
+		fork_id: &Option<String>,
```

Comment:
```suggestion
		fork_id: Option<&str>,
```

Same everywhere else.

Reply:
Done